src/main/cpp/mechanisms/DragonTale/DragonTale.cpp

```diff
@@ -1186,10 +1186,10 @@ void DragonTale::SetAlgaeReefPosition()
 	units::length::inch_t algeHeight = isBlue ? m_grabAlgaeLow : m_grabAlgaeHigh;
 	units::angle::degree_t algeAngle = isBlue ? m_grabAlgaeLowAngle : m_grabAlgaeHighAngle;
 	// Adjust the angle to the nearest 60-degree increment
-	auto info = (DragonTargetFinder::GetInstance()->GetPose(DragonTargetFinderTarget::CLOSEST_REEF_ALGAE));
-	if (info)
+	auto info = (ReefHelper::GetInstance())->GetClosestReefTagPose();
```

Comment:
Couldn't DriveToReef just publish the tag it is driving to and this subscribe to it?